.github/workflows/e2e.yml

```diff
@@ -0,0 +1,25 @@
+name: agent-test-tool
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+
+jobs:
+  e2e-rust:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Prepare service container
+        run: docker-compose -f docker-compose.e2e.yml up --build -d
+      - name: Run e2e
+        run: |
+          pip3 install --upgrade pip
+          pip3 install setuptools
+          pip3 install -r requirements.txt
+          python3 tests/e2e/run_e2e.py --expected_file=tests/e2e/data/expected_context.yaml --max_retry_times=3 --target_path=/ping
```

Comment:
I can't find the `tests/e2e/data/expected_context.yaml`. How does this test pass? Could you check the logs?
Is this a `run_e2e.py` bug or something?

Reply:
It's already in the codebase here: https://github.com/apache/skywalking-rust/blob/1465d9fc6811dcb009e75f5c1dd2d5b1ef7a52a9/tests/e2e/data/expected_context.yaml

I don't add a new test case but according to your suggestion https://github.com/apache/skywalking-rust/pull/9#issuecomment-1008699575, enable the existing but not in regression e2e test - it's a plugin test based on agent-test-tool.